Microsoft.Packaging.Tools/tasks/Trimming/NuGetPackageNode.cs

```diff
@@ -28,6 +28,7 @@ public NuGetPackageNode(string id, string version, IEnumerable<string> dependenc
         public bool IsIncluded { get; set; }
         public string Version { get; }
         public bool IsProject { get; }
+        public bool IsMetaPackage { get { return Files.Count == 0; } }
```

Comment:
the term meta-package might become a little overloaded now that we are going to have files in them. Perhaps you call this empty packages or something like that to represent they have no files.